pkg/services/action.go

```diff
@@ -324,9 +317,7 @@ func NewSubmitBusinessCase(
 		}
 
 		// TODO - EASI-2363 - rework conditional to also trigger on publishing finalized system intakes
-		// need to check intake.Status, *not* businessCase.SystemIntakeStatus - intake is what gets returned from calling updateIntake()
-		if intake.Status == models.SystemIntakeStatusBIZCASEDRAFTSUBMITTED ||
-			intake.Step == models.SystemIntakeStepDRAFTBIZCASE {
+		if intake.Step == models.SystemIntakeStepDRAFTBIZCASE {
```

Comment:
double check

Reply:
this needs to be looked at as it changes a conditional

Reply:
I believe this is right.

Reply:
Same here. Resolving!